LiskArchive/lisk-elements
# Update transaction to hold blockchain related property

### What was the problem?
Transaction was not able to hold blockchain related properties.

### How did I fix it?
Add blockchain properties in constructor, and toJSON

### How to test it?
instantiate the transaction with `blockId`, `height` and `condirmations`, and call toJSON to make sure it holds the data

### Review checklist

* The PR resolves #1197 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
